tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,6 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
```

Comment:
I'm sorry this is painful, haha! But the line you added back includes spaces. You'll need to add the newline without any trailing spaces for the diff/patch to ignore the change.

If you configure Sublime to show whitespace it's easier to detect these things. :) You may be able to configure Sublime to show changes within the editor view-- but I know Atom will do this by default. When you show staged changes it's helpful to pick up on random inserts/removes, which are usually caused by stray keystokes or missed key combinations when attempting to window-tab and such.